rest-api-spec/src/main/resources/rest-api-spec/api/cat.help.json

```
@@ -5,6 +5,9 @@
       "description":"Returns help for the Cat APIs."
     },
     "stability":"stable",
+    "headers":{
+      "accept": [ "text/plain", "application/json"]
```

```suggestion
      "accept": [ "text/plain" ]
```

Cat help looks like it only supports `text/plain`